### DISCUSSION TITLE
Old kepubify package for Calibreweb

### DISCUSSION THREAD

**Discussion by bakerboy448** [swizzin/swizzin#785] [Feature Requests] [answered]
https://github.com/swizzin/swizzin/blob/ceb08040f2ffcae8dcf2d1cd5b8b601e8eb73eae/scripts/install/calibreweb.sh#L92

Why did we hardcode to 3.1.2?  This release is over a year out of date.

The current package is 4.0.1

**Comment by brettpetch** [comment, marked as answer]
Will be looked at when https://github.com/swizzin/swizzin/pull/795 is merged.
